java/test/SpanTest.java

```diff
@@ -115,7 +115,23 @@ public ContainerSpan() {}
         private int containerSpanTableId;
         private boolean localTables = false;
 
+        /* LOGGING: Module id's for logging purposes.  Must match declarations in
```

Comment:
It would be nice if the compiler generated these.
We can probably achieve this using an annotation, e.g., @moduleid.

Reply:
This whole thing is just a library; compiler knows nothing about it.  One could declare those functions `extern` and define them in Rust and with some effort even pull them from Java. But this is probably not worth the effort.

Reply:
But the compiler could know about the annotation. We define it.